cpp/src/glm/qn/glm_softmax.cuh

```diff
@@ -25,9 +25,9 @@ namespace ML {
 namespace GLM {
 namespace detail {
 using raft::ceildiv;
-using raft::myExp;
-using raft::myLog;
-using raft::myMax;
+using raft::exp;
+using raft::log;
+using raft::max;
```

Comment:
Adding onto this, I'd actually prefer we eliminate all the `using` statements here and fully qualify these functions. Definitely not a problem introduced by this PR, but it seems like a good time to fix it, since `myExp` etc. are easier strings to find/replace than `exp`.

Reply:
Agree, I didn't like this either but was in a hurry to get something working for a complete compile. Fixed now.